plot_with_bigquery/plot_statvar_time_series.ipynb

```diff
@@ -0,0 +1,1243 @@
zCYgCmjv2mGtXe+bujIP+L4xJtRamw38EngDeA3YCrhP9kmttbcAI/Gm2zcO+CpEzhz93XMuYCnwSd/fHzXGXDjQJzPGjMT7plzBqwSifl/jwPuLD6DZWtvVA8GFt6LnfWvtOcBajii3P1HGmE/hrYC49xTHLnIm6u+eexjvD5Qbgd8D79Pj/aO19lJgBBACXHCSz/l14MPW2nTg78BvB3IBIsOZrzrnWeDOHtV1APgqyQd15oWv3944a+1zg/k8IoNBgVAfrLV7rLWXWGvn4P0N6f4+jsnGO0d0mu/jv1lr51hrlwE1wD5fo7Ou5pxfBIrp/ZvYdN+2nud14y1T7LNhp8jZ6Bj3XBHwrrW20jfd5FXgHLz3Tc8pJ+lA8Uncc7OB8UCuMeYAEG6MyR3ESxQZNk7gNe5jvu1dqvD+xvM/vo//DZxjjHH2uN/uPt7zGmMuwtuL6EprbduAL0TkDNHfPWet7bTWft3Xu+sqIBbYd8RjW4EXgKtO9DXOGJMEzLTWrvdtfwpYPKgXKTJEjDFBeMOgf1lru16nynztBfD9XX6c0/R3Ly3occ9d2d9xwCJgru895WpggjFm1cCuTOT0cA31AIYjY0yytbbc19jvh8ADvu1jgUJrbafxrpAyCThwxGNG4e0ftNBaWwvM6nFeF5DlO08x3jfdn/D1DRpnrc31/ftKYM/pul6RodbfPQe8DnzHNxWzHTgP+J21ttQYU2+MWYi3NPgzwB+ttYWcwD1nrd0FpPY4rtFaqxVYJCAc437Dt+0G4NyubdZaa4x5CW9T97eBC4Hdvl9gzOIEGGNmA38BLvP1ShEJGMd4XxkOGGttkzHmYqDTWrvbV+0Q5XutcwEfAd470dc4vL+YjDHGTLDW7sPbsDr79F2xyOnh+7npb0C2tbZnFdyLwE14p0rehDdUPZYN9P9+sec9Fw/c06OR9CXA96211XgbuXdNXXvZWrt8QBcncpoEfCBkjHkC75vcRN8867uASGPM7b5D/oO31Ba8U1a+Z4zpADzAl3s01nzWGJOAt0nZ7b4wqBdfkPQVvD/kOoGHrbW7fG8QHjXeZUgN3jnlXxqEyxUZcidzz1lra4wxv8X7Qm2BV621r/iO+zLe1VzC8PYjOaonSX/33CBdmsiwc5KvcQDL8P7iI++IU30X+Kcx5vdABXBLP8/3K7zVLG9uAAAFMUlEQVQ/kIb7nu8ha+1P8E4RiwT+7X3/ToG19sqBX6HI8HKS91wy8LoxxoP3h9BP+7ZHAC8aY0LwVvOvpEdw2+VYr3HGmM/jfW/qwRsQfdbf1yoyDCzBe9/sMMZ0rYr5A7xB0NPGmP/f3v2G6l3WcRx/f3CKznJYLF0PZE9iNHWd2Ny0ZqDtUQxpemjqMDZCiEpTH0iPVNRySpB/hkWpbM0hGzPDhqClW+k003KNORNBENGe9EeU4Qz124PfdfS3g562o8vjfu8X3HDd19/v776f3HzPdV3nW8ALdH/oIMnxdEc0jwHeTnIJMLeqXt2f34tV9a8k19D9LgW4uiWDpI+tdMcqJUmSJEmSNBTeISRJkiRJkjQwJoQkSZIkSZIGxoSQJEmSJEnSwJgQkiRJkiRJGhgTQpIkSZIkSQNjQkiSJEmSJGlgTAhJkqQpK8nsJLumQBwjSb7We39Wkh8cxPXWJhn9H31WJvnswYpBkiQd2kwISZKkQUkybRLDRoB3EkJVdW9Vrf7wopqUlYAJIUmSNCkmhCRJ0pSR5LIku9rrklY9LcmGJM8k2Zxkeuu7OsnuJDuT/LjVzUxyd5In2uvLrf6qJOuTbAfWJ/ljkhN7625LsiDJwiSPJXkqyaNJ5iQ5ArgaWJ5kR5LlbXfOmjZ2dpKHWhwPJjmh1a9NcnOb5/mJdvyksybJs0l+B3ym13ZFe5ZdSX7e+o4CC4ANLaajksxP8vskf05yf5JZH+JXI0mSDjEmhCRJ0pSQZD6wClgEnApcCBwLzAFurarPA68C30nyaWAZcGJVzQOubdPcBPykqk4BzgFu6y0xF1hSVecBG4FvtHVnAbOq6kngb8DpVfVF4ArgR1X1n1beWFUjVbVxXOi3AOtaHBuAm3tts4DFwFJgoh1Fy9pzzgW+CXyp17amqk6pqpOAo4ClVbUZeBJYUVUjwJstjtGqmg/cAfxwgvUkSdLATWbLtCRJ0sGwGLinqvYAJPkVcDrwYlVtb33uBC4GbgT2Arcn2QJsae1LgLlJxuY8JsknWvneqnq9lTcBDwBX0iWGNrf6GcC6JJ8DCjh8P+I+DTi7ldcDN/Tafl1VbwO7kxw3wRxfAe6qqreAl5M81Gs7I8nlwHTgU8DTwG/GjZ8DnAT8tj37YcDf9yN2SZI0UCaEJEnSVFfj31fVm0kWAl8FRoHvAWfS7X4+tar29ge0JMme3gQvJflnknnAcuDbrekaYGtVLUsyG9j2AWN/ox/GgQ5OciRwK7Cgql5MchVw5Ht1BZ6uqtMmFaUkSRocj4xJkqSp4mHg60mmJzma7hjVw8AJScYSHecDj7RdPzOq6j7gUuALrf0B4KKxCZOMTLDeRuDyNs/OVjcDeKmVV/b6vgZ88n3meRQ4t5VXtJgP1B/o7ig6rB1hO6PVjyV//tGeuX8PUT+mZ4GZY59TksP7dyRJkiSNZ0JIkiRNCVX1F2At8Cfgcbr7f/5Nl+z4bpJn6O4U+ildImRLkp3AI8BlbZqLgQXtgufdvLvz571spkvkbOrV3QBcl+Qp9t1JvZXuKNqOJMvHzXMRsKrFcgHw/QN68M49wHPAbuCXwGMAVfUK8AtgF3A/8ERvzFrgZ0l20B0RGwWuT/JXYAf73kMkSZK0j1SN34UtSZIkSZKkQ5k7hCRJkiRJkgbGS6UlSZL+T5KcTPefyPreqKpFH0U8kiRpuDwyJkmSJEmSNDAeGZMkSZIkSRoYE0KSJEmSJEkDY0JIkiRJkiRpYEwISZIkSZIkDcx/Adyt4fjzz4vtAAAAAElFTkSuQmCC\n",
```

Comment:
Can you check in the code and text only instead of the actual content like the image here?

Reply:
Replaced with .py